### ISSUE TITLE
Types not working with Nuxt

### ISSUE THREAD

**Issue by antfu** [antfu/unplugin-auto-import#1]
_Originally posted by @wobsoriano in https://github.com/antfu/vue-global-api/issues/20#issuecomment-904510072_

```javascript
buildModules: [
  '@nuxt/typescript-build',
  '@nuxtjs/composition-api/module',
  'nuxt-typed-vuex',
  [
    'unplugin-auto-import/nuxt',
    {
      // global imports to register
      // imports: ['vue2']
      imports: [
        {
          '@nuxtjs/composition-api': CommonCompositionAPI,
        },
      ],
    },
  ],
],
```

Preset `vue2` and custom preset still gives cannot find name 'defineComponent' etc..

![Screen Shot 2021-08-24 at 6 12 21 PM](https://user-images.githubusercontent.com/13049130/130599213-5cbbe09a-db38-4d1e-a499-56f0c6c9f88c.png)

Also tried restarting vs code with no luck

**Comment by antfu** [comment]
@wobsoriano Do you see `auto-imports.d.ts` generated under the root?

**Comment by wobsoriano** [comment]
Running `yarn dev` generates `auto-imports.d.ts`

```typescript
// Generated by 'unplugin-auto-import'
declare global {

}
export {}
```

Then I get errors in console ` ERROR  Cannot read property 'module' of undefined`

**Comment by antfu** [comment]
Should be fixed in v0.1.1
